renovate.json

```diff
@@ -41,12 +41,17 @@
     }
   ],
   "packageRules": [
-    { 
```

Comment:
thoughts on moving this to common instead of just removing? I think in most cases we would want renovate to pin digests for us if something was added without one.